### ISSUE TITLE
Module is not available in Backend when calendar is not installed

### ISSUE THREAD

**Issue by CMSworker** [inspiredminds/contao-sibling-navigation#11]
Problem description: When the calendar extension is not installed you can't create a siblings module for news.

First test (base):
- Installed Contao 4.12.5 (core bundle with news and comments only, calendar not installed)
- Installed contao-sibling-navigation via Contao Manager
- Now wanted to create the news-siblings-module, but it is not shown/available in the select list "Modul type"

Second test:
- Install the Contao Core Calendar extension via Contao Manager
- Now the siblings modules are shown/available in the module type select list in backend

Third test:
- Remove calendar extension again
- Siblings module is not available anymore in the module type list in backend

**Comment by fritzmg** [comment]
Hm, I could not reproduce. With only the following present in the `composer.json`:

```json
{
    "name": "contao/managed-edition",
    "type": "project",
    "require": {
        "contao/conflicts": "@dev",
        "contao/core-bundle": "^4.12",
        "contao/manager-bundle": "4.12.*",
        "contao/news-bundle": "^4.12",
        "inspiredminds/contao-sibling-navigation": "^1.3"
    },
    "extra": {
        "contao-component-dir": "assets"
    },
    "scripts": {
        "post-install-cmd": [
            "@php vendor/bin/contao-setup"
        ],
        "post-update-cmd": [
            "@php vendor/bin/contao-setup"
        ]
    }
}
```

the module _News sibling navigation_ is available for selection in the back end.

**Comment by CMSworker** [comment]
Tried it with a new installation, same problem. Maybe it's related to the order of installing everything with Contao Manager? Here more detailed step-by-step:

1. Download _contao-manager.phar_ from contao.org, save in _/web/_ directory, add _.php_ at the end
2. Open Contao Manager in browser, add user account
3. Install Contao, minimal version (just core)
![screenshot1](https://user-images.githubusercontent.com/31544192/147954010-27ad2a47-4b37-45c9-9a6d-4cb59e91d737.jpg)
4. Open the Contao install tool to add database infos and finish core installation
5. Back to Contao Manager: Add the Contao Core News module
![screenshot2](https://user-images.githubusercontent.com/31544192/147954126-a32c6431-b569-40b3-bc3f-f89434c60a97.jpg)
6. Open the Contao install tool again to finish the news installation (add news tables)
7. Back to Contao Manager: Add the siblings extension
![screenshot3](https://user-images.githubusercontent.com/31544192/147954264-2b5d18e9-de12-4301-bbba-1858ab62e41c.jpg)
8. Open the Contao install tool again to finish siblings installation
9. Login to Contao backend, create a new Theme (I just named it "Test" and didn't add anything to it for testing)
10. Go to the theme's module page, create a new module -> it is not possible to select the siblings navigation, it is missing in the dropdown list
![screenshot4b](https://user-images.githubusercontent.com/31544192/147955577-f96c6caa-1062-48ae-9e46-db0e74b0c3b6.jpg)

It is also not visible when you create a test news archive and some test news entries.

The composer.json is looking like this after installation as described:
```

{
    "type": "project",
    "require": {
        "contao/conflicts": "*@dev",
        "contao/manager-bundle": "4.12.*",
        "contao/news-bundle": "4.12.*",
        "inspiredminds/contao-sibling-navigation": "^1.3"
    },
    "extra": {
        "contao-component-dir": "assets"
    },
    "scripts": {
        "post-install-cmd": [
            "@php vendor/bin/contao-setup"
        ],
        "post-update-cmd": [
            "@php vendor/bin/contao-setup"
        ]
    }
}
```

**Comment by fritzmg** [comment]
Please test `dev-master`, may be that will fix it.

**Comment by CMSworker** [comment]
Yes that seems to fix it.

![screenshot-7](https://user-images.githubusercontent.com/31544192/147959937-2c81001c-c570-45a3-9823-97bde4424b25.jpg)

![screenshot-8](https://user-images.githubusercontent.com/31544192/147959942-d4d3a07c-c4ea-4dda-8edf-3335cee52aff.jpg)

Before I also made a test with installing Contao full and than removing Calendar etc. and adding the siblings extension (1.3.1), this way the module is also not selectable in the dropdown list (probably not relevant anymore after the fix)

![screenshot1](https://user-images.githubusercontent.com/31544192/147960111-c65862ff-abcd-49b1-ae5e-15c782d78722.jpg)

![screenshot2](https://user-images.githubusercontent.com/31544192/147960120-a1046920-0285-4607-8565-a38bf88cded4.jpg)

![screenshot3](https://user-images.githubusercontent.com/31544192/147960133-2b47e6b7-2bb1-4c48-a6ab-5c0429e18feb.jpg)

![screenshot4](https://user-images.githubusercontent.com/31544192/147960139-dace239f-efaa-4f46-b4cf-11a23d89a9c9.jpg)

![screenshot4b](https://user-images.githubusercontent.com/31544192/147960176-6c3fc546-8163-4929-8b56-8bfda3f21dff.jpg)

Thank you for the extension and your support!

**Comment by CMSworker** [comment]
Just repeated my first test (install Contao minimal, add news, than add siblings extension), but with siblings "dev-master". It works, the module is available in the module select list in backend.

**Comment by fritzmg** [comment]
Released as `1.3.2`.
